learn/cookbooks/aws.md

```diff
@@ -36,72 +36,72 @@ Select the specifications of the server you want Meilisearch to run on.
 We recommend prioritizing memory allocation for better Meilisearch performance.
 
 ::: tip
-The **free tier** is sufficient for tests or prototypes, but not recommended for large datasets.
+The "free tier" is sufficient for tests or prototypes, but not recommended for large datasets.
```

Comment:
How about we leave it without quotes? This way we are referring to whatever free tier Amazon has and don't need to bother with its proper name.